Source/relay/TourGuide/Sets/Daily Dungeon.ash

```diff
@@ -14,7 +14,10 @@ void LockPickingGenerateTasks(ChecklistEntry [int] task_entries, ChecklistEntry
 
 void SDailyDungeonGenerateTasks(ChecklistEntry [int] task_entries, ChecklistEntry [int] optional_task_entries, ChecklistEntry [int] future_task_entries)
 {
-	
+	if (my_path().id == PATH_COMMUNITY_SERVICE) {
```

Comment:
while we're here, are there any other paths we can return the tile on? ed the undying is one, i feel like there's 1-2 others but they aren't coming to mind. maybe we just add actually ed here and call it a day.

Reply:
Added Ed to the conditional too. I skimmed a bunch of other paths' wiki pages and didn't find any others, but easy enough to add later if we find more.